From 62d93d3f9a8d72ee3ded8d7fb4eb7045b727e752 Mon Sep 17 00:00:00 2001
From: Craig Kewley <craigx.kewley@intel.com>
Date: Mon, 25 Jul 2016 14:15:24 +0100
Subject: [PATCH 41/96] ASoC: Intel: Boards: Add SND_SOC_COMPRESS dependency

SND_SOC_COMPRESS was moved out to an optional config
option, enable it when the GP MRB machine driver is
selected.

Signed-off-by: Craig Kewley <craigx.kewley@intel.com>

Conflicts:
	sound/soc/intel/Kconfig
---
 sound/soc/intel/boards/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/intel/boards/Kconfig b/sound/soc/intel/boards/Kconfig
index 53a07f4..6a6995e 100644
--- a/sound/soc/intel/boards/Kconfig
+++ b/sound/soc/intel/boards/Kconfig
@@ -378,6 +378,7 @@ config SND_SOC_INTEL_BXT_GPMRB_MACH
 	tristate "ASoC Audio driver for BXT IVI GP MRB"
 	select SND_SOC_INTEL_SST
 	select SND_SOC_TDF8532
+	select SND_SOC_COMPRESS
 	default n
 	help
 	   This adds support for ASoC machine driver for Broxton IVI GP MRB platform
-- 
1.9.1

